app/scripts/directives/mobileClientBuildsList.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+(function() {
+  angular.module('openshiftConsole').component('mobileClientBuildsList', {
+    controller: [
+      '$timeout',
+      '$anchorScroll',
+      '$location',
+      'APIService',
+      'DataService',
+      MobileClientBuildsList
+    ],
+    bindings: {
+      mobileClient: '<'
+    },
+    templateUrl: 'views/mobile-client-builds-list.html'
+  });
+
+  function MobileClientBuildsList(
+    $timeout,
+    $anchorScroll,
+    $location,
+    APIService,
+    DataService
+  ) {
+    var ctrl = this;
+    var buildConfigsVersion = APIService.getPreferredVersion('buildconfigs');
+    var watches = [];
+    var anchor;
+
+    ctrl.$onChanges = function(changes) {
+      var mobileClientChanged = _.get(changes, "mobileClient.currentValue", false);
```

Comment:
Change `"mobileClient.currentValue"` to `'mobileClient.currentValue'` to keep quotes consistent.